Examples/Io/Json/src/JsonGeometryList.cpp

```diff
@@ -67,6 +69,11 @@ std::vector<Acts::GeometryIdentifier> ActsExamples::readJsonGeometryList(
     const std::string& path) {
   nlohmann::json data;
   std::vector<Acts::GeometryIdentifier> geoIdList;
+  namespace fs = std::filesystem;
+  if (!fs::exists(fs::status(fs::path(path)))) {
+    throw fs::filesystem_error(
+        path, std::make_error_code(std::errc::no_such_file_or_directory));
+  }
   std::ifstream infile(path, std::ifstream::in | std::ifstream::binary);
```

Comment:
```suggestion
  std::ifstream infile(path, std::ifstream::in | std::ifstream::binary);
  if (!infile. good()) {
    throw fs::filesystem_error(
        path, std::make_error_code(std::errc::no_such_file_or_directory));
  }
```
I believe that is slightly easier